pkg/rtc/clientinfo.go

```diff
@@ -92,13 +92,9 @@ func (c ClientInfo) ComplyWithCodecOrderInSDPAnswer() bool {
 	return !((c.isLinux() || c.isAndroid()) && c.isFirefox())
 }
 
-// Rust SDK can't decode unknown signal message (TrackSubscribed and ErrorResponse)
-func (c ClientInfo) SupportTrackSubscribedEvent() bool {
```

Comment:
we cannot remove this. existing versions of Rust is still out there that will break as soon as this is shipped.

the protocol version on the Rust side need to be bumped to signal support for this protocol level.